README.adoc

```diff
@@ -55,55 +71,44 @@ Target repositories are configured in `repositories.yaml` in the following form:
 |===
 
 
-
 These are the only variables currently being used on the playbooks, but can be extended easily as more are needed.
 
 NOTE: If a new variable is introduced, it needs to be added to all repository objects!
 
-==== Templating
-The folder and file structure under `template` is considered as source structure which will be synchronized across all repositories in the following manner:
+Additional settings can be found in `playbook/group_vars/all`, but these are not intended to be freely changed and should be treated with care.
 
-* Regular files and folders are simply copied
-* Files with an extension of `.j2` will be processed as jinja2 templates and copied to the target repositories with the `.j2` extension removed. The default jinja2 variable delimiter has been replaced with `{[ }]` since a lot of the template files contain `{{  }}` and caused issues.
-* File and directory names which contain `\[[product]]` will have this substituted for the value of the `product_string` variable. For example: "stackable-\[[product]]-operator.service" would become `stackable-kafka-operator.service`.
 
-To remove files or directories that already exist in the target repositories these need to be configured in `repositories.yaml` under the `retired_files` key.
+== Making changes to the template
 
-Anything that is listed here will be deleted from the target repositories.
+If you want to make a change that should be rolled out to all operators, make the change in the `template` directory.
+Consult the section above to learn more about the structure of the template.
 
-NOTE: Deletion is the last step that is performed, so if there is an overlap between files existing in the template folder and this setting, the files would not be rolled out, since they'd get deleted before creting the pull request.
+=== Test changes locally
 
-=== Local
-While this can in theory also be used locally this has not yet been tested and may or may not be documented and supported here in the future.
+1. Create the directory `work`, clone all (or some) operators into this directory, and run the `test.sh` script.
```

Comment:
I believe it will clone the repos for you. So you only have to adjust the list in the repositories.yaml.

Reply:
Oh yeah right. Lemme check and then I'll update this (it was copy pasted from other seemingly slightly outdated instructions)

Reply:
@NickLarsenNZ I've changed it now!